include/aws/http/connection.h

```diff
@@ -548,6 +546,31 @@ int aws_http2_connection_get_received_goaway(
     uint32_t *out_http2_error,
     uint32_t *out_last_stream_id);
 
+/**
+ * Increment the connection's flow-control window to keep data flowing (HTTP/2 only).
+ *
+ * If the connection was created with `manual_window_management` set true,
+ * the flow-control window of the connection will shrink as body data is received for all the streams created on it.
+ * (headers, padding, and other metadata do not affect the window).
+ * The initial connection flow-control window is 65,535.
+ * Once the connection's flow-control window reaches to 0, all the streams on the connection stop receiving any further
+ * data.
+ *
+ * If `manual_window_management` is false, this call will have no effect.
+ * The connection maintains its flow-control windows such that
+ * no back-pressure is applied and data arrives as fast as possible.
+ *
+ * If you are not connected, this call will have no effect.
+ *
+ * Fails when the input is invalid. The limit of the Maximum Size is 2147483647. If the increment size cause the
```

Comment:
nit: Most programmers would eyeball 2147483647 and know it's  the maximum value of 32-bit signed integer, but it's clearer to write it out or in the form `2**31 - 1`